pkg/mattermost/mattermost.go

```diff
@@ -0,0 +1,208 @@
+package mattermost
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/infracloudio/botkube/pkg/config"
+	"github.com/infracloudio/botkube/pkg/execute"
+	"github.com/infracloudio/botkube/pkg/logging"
+	"github.com/mattermost/mattermost-server/model"
+)
+
+var client *model.Client4
+var webSocketClient *model.WebSocketClient
+var botUser *model.User
+var botTeam *model.Team
+var botChannel *model.Channel
+
+const (
+	// BotName stores Botkube details
+	BotName = "botkube"
+	// ChannelPurpose describes the purpose of Botkube channel
+	ChannelPurpose = "Botkube alerts"
+	// WebSocketProtocol stores protocol initials for web socket
+	WebSocketProtocol = "ws:"
+	// Logs file name
+	Logs = "logs"
```

Comment:
@mugdha-adhav why do we need this?

Reply:
I am assuming this is the name of file if we are returning response as a file?

Reply:
Updated the logs filename.